pkg/game/scoring_mid_game_test.go

```diff
@@ -0,0 +1,106 @@
+package game
+
+import (
+	"testing"
+
+	"github.com/YetAnotherSpieskowcy/Carcassonne-Engine/pkg/deck"
+	"github.com/YetAnotherSpieskowcy/Carcassonne-Engine/pkg/game/elements"
+	"github.com/YetAnotherSpieskowcy/Carcassonne-Engine/pkg/game/position"
+	"github.com/YetAnotherSpieskowcy/Carcassonne-Engine/pkg/stack"
+	"github.com/YetAnotherSpieskowcy/Carcassonne-Engine/pkg/tiles"
+	"github.com/YetAnotherSpieskowcy/Carcassonne-Engine/pkg/tiles/feature"
+	"github.com/YetAnotherSpieskowcy/Carcassonne-Engine/pkg/tiles/side"
+	"github.com/YetAnotherSpieskowcy/Carcassonne-Engine/pkg/tiles/tiletemplates"
+	"github.com/YetAnotherSpieskowcy/Carcassonne-Engine/pkg/tilesets"
+)
+
+/*
+|            -1   0    1
+|               | @ |
+|               .\ /.
+|1              ..4..
+|               ./ \.
+|               |   |
+|     ..!.......|   |.....
+|     ...........\ /..[!].
+|0    --3----2-@--0---[1].
+|     ..|.............[ ].
+|     ..|.................
+*/
+func TestScoringMidGame(t *testing.T) { // nolint: gocyclo
```

Comment:
This test seems quite repetitive, do you think you could minimize repetition here somehow?

Reply:
little simplified